cmd/api-server/applications/applications.go

```diff
@@ -23,6 +23,12 @@ func (a *ApiHandler) ListApplications(c *gin.Context) {
 		c.AbortWithStatus(http.StatusNotFound)
 		return
 	}
+	if org.AccountID != c.MustGet("currentUser").(models.User).ID {
```

Comment:
The user_id should not be used to filter the data ownership; with this, the system will be limited to only one user. There's a field in the user model called `client` which can be used to "group" users under one client